etna/ciim/client.py

```diff
@@ -294,7 +397,11 @@ def search(
                 created_end_date, supplementary_time=time.max
             )
 
-        return self.make_request(f"{self.base_url}/data/search", params=params).json()
+        resp = self.make_request(f"{self.base_url}/data/search", params=params).json()
+        filter_aggregations_response, results_response = resp["responses"]
+        return self.resultlist_from_response(
+            results_response, filter_aggregations_response.get("aggregations")
+        )
```

Comment:
```suggestion
        resp = self.make_request(f"{self.base_url}/data/search", params=params).json()
        _, results_response = resp["responses"]
        return self.resultlist_from_response(
            results_response, results_response.get("aggregations")
        )
```

Reply:
@ababic , there is an issue when passing `filter_aggregations_response.get("aggregations")` as the checkbox values in the `Refine by results` shore more than desired, response  depends on `results_response.get("aggregations")` the second part of the api response.
Ex url: /search/catalogue/?filter_keyword=ufo&opening_start_date_0=&opening_start_date_1=&opening_start_date_2=&opening_end_date_0=&opening_end_date_1=&opening_end_date_2=&q=royal&per_page=20&sort_order=asc&display=list&group=tna

Obviously the impact of using `ResultList.filter_aggregations` may change. You might want to revisit this part.

Reply:
@TNA-Allan I believe this is now resolved. `ResultList.filter_aggregations` has been replaced by `ResultList.bucket_counts`, which is populated from `ResultList.aggregations["groups"]["buckets"]` for all cases except for `search()`, which instead takes `["aggregations"]["groups"]["buckets"]` value from the first/separate ES response. This reflects the old behaviour from [BucketsMixin.get_group_buckets()](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/search/views.py#L66-L77), which can now be removed.

Reply:
@ababic , this update looks good in terms of accessing the first/separate ES response, but the above url run does not show the filtered records (169) for `collection` and other attributes which uses the second ES response for `search`. This [filter_aggregations](https://github.com/nationalarchives/ds-wagtail/blob/bfb2e907bb81e4da5a308cba95dcab4c251a7623/etna/search/views.py#L455) attributes looks missed replacing with `aggregations`

Reply:
Good catch @TNA-Allan! I thought I was going crazy there for a minute, as the failure was completely silent. I'll take that as a good lesson to be more cautious about wrapping things in `try/except` temporarily. Fixed in [984e97a](https://github.com/nationalarchives/ds-wagtail/pull/422/commits/984e97a6cd2c0785825a4a161ce3f03177a09ace)